Facebook.AccountKit.iOS/nuget/Xamarin.Facebook.AccountKit.iOS.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.Facebook.AccountKit.iOS</id>
+    <title>Facebook Account Kit SDK for Xamarin iOS</title>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>C# bindings for Facebook Account Kit's official SDK</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=865579</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=865583</licenseUrl>
```

Comment:
Please, update the license URL to this one: https://go.microsoft.com/fwlink/?linkid=873258